setup.py

```diff
@@ -3,9 +3,9 @@
 
 
 def supports_av():
-    h = Popen("ldconfig -p | grep toxav", shell=True, stdout=PIPE)
+    h = Popen("ld -ltoxav", shell=True, stderr=PIPE)
     out, err = h.communicate()
-    return 'toxav' in str(out)
+    return 'toxav' not in str(err)
```

Comment:
On Mac os, the output is like this:

    $ ld -ltoxxav
    ld: library not found for -ltoxxav

I agree this is not a good solution for all Python extensions, but specifically for toxav it should be because the likelihood of having toxav as an error message word is very low.